mlflow/pyfunc/__init__.py

```diff
@@ -1260,6 +1257,22 @@ def spark_udf(
         is_spark_in_local_mode or should_use_nfs or is_spark_connect
     )
 
+    # For spark connect mode,
+    # If client code is executed in databricks runtime and NFS is available,
+    # In driver side, we save model to NFS temp directory,
+    # then in executor side we can load model from the NFS temp directory.
```

Comment:
```suggestion
    # we save model to NFS temp directory in the driver
    # and load the model in the executor.
```

not sure if this makes sense.